main.go

```diff
@@ -79,32 +79,29 @@ func main() {
 	// ********************************************************************************
 	// Configure open tracing
 	// ********************************************************************************
-	var span opentracing.Span
 	// Enable Jaeger
-	if jaeger.IsOpentracingEnabled() {
-		jaegerCloser := jaeger.InitJaeger("nsc")
-		defer func() { _ = jaegerCloser.Close() }()
-		span = opentracing.StartSpan("nsc")
-	}
-	cmdSpan := spanhelper.NewSpanHelper(ctx, span, "nsc")
+	jaegerCloser := jaeger.InitJaeger("nsc")
+	defer func() { _ = jaegerCloser.Close() }()
 
+	cmdSpan := spanhelper.FromContext(ctx, "Init nsc")
```

Comment:
Please don't.  Use log.Entry(ctx) for this sort of logging.

Reply:
Used `log.Entry(ctx)` for nsc initialization logs